File: ap/codeforces/cf_ed62/SolutionB.java
//
// https://codeforces.com/contest/1140/problem/B
// Author: montreal91
// Type: training
// Time: 00:20
// Failed attempts: 0
//
import java.util.Scanner;

public class SolutionB {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int T = in.nextInt();
        int n;
        String s;
        for (int i=0; i<T; i++) {
            n = in.nextInt();
            s = in.next();
            System.out.println(solveString(s));
        }
    }

    private static int solveString(String s) {
        return Math.min(findLeft(s), findRight(s));
    }

    private static int findLeft(String s) {
        int i = 0;
        for (i=0; i<s.length(); i++) {
            if (s.charAt(i) == '>') {
                return i;
            }
        }
        return i;
    }

    private static int findRight(String s) {
        int i=0;
        for (i=0; i<s.length(); i++) {
            if (s.charAt(s.length() - i - 1) == '<') {
                return i;
            }
        }
        return i;
    }
}

File: ap/codeforces/cf693/solution_b.cc

//
// Problem: https://codeforces.com/contest/1472/problem/B
// Author: montreal91
// Type: contest
// Time: 00:22
// Failed attempts: 0
//

#include <algorithm>
#include <chrono>
#include <cmath>
#include <cstdint>
#include <iostream>
#include <fstream>
#include <limits>
#include <map>
#include <string>
#include <unordered_map>
#include <utility>
#include <vector>

#ifndef ONLINE_JUDGE
  #include "util.h"
#endif // ONLINE_JUDGE

using namespace std;


bool is_good(vector<int>& cands) {
  int ones = 0, twos = 0;
  for (int c : cands) {
    if (c == 1) {
      ones++;
    }
    else if (c == 2) {
      twos++;
    }
  }

  // cout << "ONES: " << ones <<endl;

  if (ones % 2 != 0) {
    return false;
  }
  if (ones == 0 && twos % 2 != 0) {
    return false;
  }
  return true;
}

string verbose(bool b) {
  if (b) {
    return "YES";
  }
  return "NO";
}


void function(istream& in, ostream& out) {
  ios::sync_with_stdio(false);
  in.tie(nullptr);
  int t, n;
  in >> t;
  vector<int> cands;

  for (int i=0; i<t; i++) {
    in >> n;
    cands.resize(n);

    for (int j=0; j<n; j++) {
      in >> cands[j];
    }
    // debug_print(cands);
    out << verbose(is_good(cands)) << endl;
  }
}

int main() {
  #ifndef ONLINE_JUDGE
    using namespace chrono;
    auto time1 = duration_cast<milliseconds>(
      system_clock::now().time_since_epoch()
    );
  #endif // ONLINE_JUDGE

  function(cin, cout);

  #ifndef ONLINE_JUDGE
    auto time2 = duration_cast<milliseconds>(
      system_clock::now().time_since_epoch()
    );
    cout << "Time consumed: " << milliseconds(time2 - time1).count();
    cout << " ms.\n";
  #endif // ONLINE_JUDGE

  return 0;
}
